partiql-lang/src/main/kotlin/org/partiql/lang/eval/EvaluatingCompiler.kt

```diff
@@ -243,6 +253,10 @@ internal class EvaluatingCompiler(
                             ?: ExprValue.nullValue
                 }
             }
+            val everyAccFunc: (ExprValue?, ExprValue) -> ExprValue = { accumulated, nextItem ->
+                checkIsBooleanType("EVERY", nextItem)
+                accumulated?.let { ExprValue.newBoolean(it.booleanValue() && nextItem.booleanValue()) } ?: nextItem
+            }
```

Comment:
I know performance isn't exactly a concern, but perhaps we use ExprAggregator so that we don't have to instantiate an ExprValue for every accumulated row just to be unpacked in the next accumulator.next call.

```kotlin
val everyAccFunc = { filter: (ExprValue) -> Boolean ->
    object : ExprAggregator {

        var everyIsTrue: Boolean = true

        override fun next(value: ExprValue) {
            if (value.isNotUnknown() && filter.invoke(value)) {
                checkIsBooleanType("EVERY", value)
                everyIsTrue = everyIsTrue && value.booleanValue()
            }
        }

        override fun compute() = ExprValue.newBoolean(everyIsTrue);
    }
}
```